pwa/actions/activation.ts

```diff
@@ -0,0 +1,3 @@
+export class Activation {
```

Comment:
Still used?

Reply:
Yes in pages/activation/index.tsx

Reply:
This class isn't used, the one from `user.ts` yes.